scripts/phpunit/bootstrap.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+// Compatibility for PHPUnit < 6 and PHPUnit 6+.
+if (class_exists('PHPUnit_Framework_TestSuite') === true && class_exists('PHPUnit\Framework\TestSuite') === false) {
```

Comment:
Is `===` necessary? I think `class_exists()` always returns a boolean. You should be fine with:
```
if (class_exists('PHPUnit_Framework_TestSuite') 
  && !class_exists('PHPUnit\Framework\TestSuite')) {
```

Reply:
I'd like a comment here describing what this condition indicates. Does this mean it's PHPUnit 4, for instance?

Reply:
> Is === necessary? I think class_exists() always returns a boolean.

Makes sense to me... I think I copied the code from a comment. I'll update.

Are you saying you'd like more detail in the comment on line 3? Based on your other comment, if we drop PHPUnit 4 support, then this file is a shim to make PHPUnit 5 work with PHPUnit 6 namespaced classes. You're just asking for that statement in a more verbose comment?